api-docs/client-files.md

```diff
@@ -0,0 +1,262 @@
+# Client files
+
+This api is provided to upload and keep track of client files.
+
+## Get a single client file
+
+```http
+GET /api/clients/:file/:id/files
```

Comment:
comunidades-unidas has RESTful APIs. The term REST is often bastardized to mean a lot of things -- it started out as an academic paper in a journal and has morphed into meaning a lot of different things. A main concept in REST, though, is [resources](https://www.tutorialspoint.com/restful/restful_resources.htm).

The way you structure your url is determined by the resources that are involved in the API. In this case, we have two resources: `clients` and `files`. The more common way to structure urls is make the url readable left-to-right where the id parameter comes after the resource name.

In other words, let's rearrange the url here:

```suggestion
GET /api/clients/:clientId/files/:fileId
```